axeDevToolsXCUI.xcframework/ios-arm64_x86_64-simulator/axeDevToolsXCUI.framework/Info.plist

```diff
@@ -41,7 +41,7 @@
 	<key>DTXcodeBuild</key>
 	<string>13F100</string>
 	<key>DequeMeta</key>
-	<string>2.8.0-867-eb5a6c00</string>
+	<string>2.8.1-879-e9cefeab</string>
```

Comment:
Is the piece after the 2.8.1 a commit number? Or is that the beta build number?

Reply:
responded privately